src/components/portal/products/ProductForm.js

```diff
@@ -207,10 +225,12 @@ class ProductForm extends Component {
               onChange={this.onChange}
             />
 
-            <VendorTypeAhead
-              placeholder="Vendor"
-              onClickHandler={this.props.Vendor_Update_TypeAhead}
-            />
+            {this.props.user.userType === 'admin' && (
```

Comment:
I'm a bit confused about the '&&' here. How is this working and what is it doing?

Reply:
It's just working the way it works